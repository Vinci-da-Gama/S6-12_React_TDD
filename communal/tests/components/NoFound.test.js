import React from 'react';
import { shallow } from 'enzyme';

import NoFoundCompo from '../../src/components/NoFound';

test('should render NoFound Page correctly.', () => {
    const wrapper = shallow(<NoFoundCompo />);
    expect(wrapper).toMatchSnapshot();
});
