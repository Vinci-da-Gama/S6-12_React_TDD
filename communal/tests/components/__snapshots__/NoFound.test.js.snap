// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoFound Page correctly. 1`] = `
<div>
  <h1>
    404
    <Badge
      color="secondary"
      pill={false}
      tag="span"
    >
      No Found.
    </Badge>
  </h1>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
